### DISCUSSION TITLE
要个激活码这么难吗

### DISCUSSION THREAD

**Discussion by xiaolin6456** [getlantern/lantern-forum-cn#757] [General]
![微信图片_20240727111127](https://github.com/user-attachments/assets/de60226a-5256-43dd-b3cf-ab51b55d35a4)
这是我的付款凭证，我的邮箱:xiaolin6456@sina.com ，一直收不到激活码，三周了已经
目前使用最新的桌面版应用程序测试版

**Comment by MaxMKMax** [comment]
@xiaolin6456 您好. 不好意思让您久等了. 非常抱歉. 我已经帮您申请激活码了. 这两天请查看这个帖子或邮箱.

**Comment by MaxMKMax** [comment]
![image](https://github.com/user-attachments/assets/409c669c-e131-43b1-8e6a-a27bbd0b633a)

@xiaolin6456 您好. 您已经是专业版了. 不需要激活码了. 早就已经登录成功. 您缺少的是 device linking code. 这是您的专属号码. 
613799

这也可以帮到您:
https://lantern.io/zh-Hans/faq
